### ISSUE TITLE
[N64] Garbage left on screen with MAME RDP

### ISSUE THREAD

**Issue by CasualPokePlayer** [higan-emu/ares#345]
This is an issue noted in the [original PR](https://github.com/ares-emulator/ares/pull/324) for MAME's RDP, although no issue was formally filed from what I can find so I'm doing it here. Note that these issues don't appear with MAME itself. Some investigation indicates that these garbage frames normally take [this path](https://github.com/ares-emulator/ares/blob/master/ares/n64/vi/vi.cpp#L97-L100) with Vulkan, indicating these frames are supposed to be blanked by some conditions set by VI registers? Code is confusing from this point, so I don't know.

~~(Note this issue doesn't occur when "disabling" Vulkan in the settings due to [this bug](https://github.com/ares-emulator/ares/issues/344), thus only occuring when leaving out Vulkan support entirely in the build)~~ not applicable since b03da384d847f743d7a99020956ac2e52b4c1b70

**Comment by rasky** [comment]
@CasualPokePlayer maybe I’m missing something but I don’t see a description of the bug. Can you please describe what is the bug, when it happens, how to reproduce it and what platforms do you see it on?

**Comment by CasualPokePlayer** [comment]
It happens on frames meant to be blank, often loading screens. Easily reproducible with Super Mario 64, where it will appear when loading the floating mario head for the "Press Start" screen.

~~As noted, this won't appear on builds with Vulkan support at all even with Vulkan disabled in settings due to the bug I linked. Any build with Vulkan left out with build flags should show the effect. I'm personally using Windows.~~

EDIT: The above bug with Vulkan settings has been fixed (b03da384d847f743d7a99020956ac2e52b4c1b70), so this can reproduced by simply disabling Vulkan in the settings.
